iceoryx_posh/source/capro/service_description.cpp

```diff
@@ -181,6 +150,38 @@ ServiceDescription::operator cxx::Serialization() const
                                       interface);
 }
 
+cxx::expected<ServiceDescription, cxx::Serialization::Error>
+ServiceDescription::deserialize(const cxx::Serialization& serialized) noexcept
+{
+    ServiceDescription deserializedObject;
+
+    using ScopeType = std::underlying_type<Scope>::type;
+    using InterfaceType = std::underlying_type<Interfaces>::type;
```

Comment:
This is not quite correct, since the scope type is actually `Scope`, so this must be either `ScopeUnderlyingType` or shorter `ScopeUT`. Since this is confined to the class, I think the abbreviation is not too confusing.

Reply:
True indeed, done.